libvirt.go

```diff
@@ -300,6 +302,47 @@ func (d *Driver) Create() error {
 	return d.Start()
 }
 
+func (d *Driver) createDisk() error {
+	u, err := url.Parse(d.DiskPathURL)
+	if err != nil {
+		return fmt.Errorf("Invalid URL: %s", d.DiskPathURL)
+	}
+
+	if u.Scheme == "file" || u.Scheme == "" {
+		// For local image on Linux, we can avoid copying by making use of base image through qemu-img
+		// Local disk image so we can avoid copying by making use of backing image feature of qcow2
```

Comment:
How so?

Reply:
I don't see much difference in meaning/information provided by
` For local image on Linux, we can avoid copying by making use of base image through qemu-img` and `Local disk image so we can avoid copying by making use of backing image feature of qcow2`
Both lines repeat "we are on linux, we have a local image, we can avoid the copy", and then you have 2 different ways of saying we are going to use an overlay instead of doing the copy.